src/components/views/rooms/RoomSettings.js

```diff
@@ -73,10 +74,11 @@ const BannedUser = React.createClass({
                 >
                     { _t('Unban') }
                 </AccessibleButton>
-                {this.props.member.userId}
+                <strong>{this.props.member.name}</strong>
```

Comment:
i'm not keen at removing the mxid from the UI, as this is the identifier which has actually been banned.